frontend/src/slices/orders/order.slice.ts

```diff
@@ -43,6 +52,22 @@ const { reducer, actions, name } = createSlice({
       })
       .addCase(calculateOrderPrice.rejected, (state) => {
         state.dataStatus = DataStatus.REJECTED;
+      })
+      .addCase(getOrder.pending, (state) => {
+        state.dataStatus = DataStatus.PENDING;
+      })
+      .addCase(getOrder.fulfilled, (state, action) => {
+        state.orders = [action.payload];
```

Comment:
Quite confusing - here we have `orders` but there always will be only 1 order.

Reply:
it's because for dashboard this state variable is used for containg all busines/driver orders, but for customer part we decided that for now there will be only one order at a time.

Reply:
so I think we need to split driver orders and customer orders.
and actually customer will have an array of orders, but the first one inside this arr will be current one

Reply:
replaced with currentOrder